docs/api/python-api.md

```diff
@@ -0,0 +1,6 @@
+# Python API
+
+```{eval-rst}
+.. automodule:: spglib.spglib
+   :members:
+```
```

Comment:
To investigate, the following should work: 
````suggestion
```{eval-rst}
.. automodule:: spglib
   :members:
```
````
I've just seen that it does indeed work in fmf:
- https://github.com/teemtee/fmf/blob/e31f7e7207c417cf84a10ff768da8e63a0ec7944/fmf/__init__.py#L6-L10
- https://github.com/teemtee/fmf/blob/e31f7e7207c417cf84a10ff768da8e63a0ec7944/docs/modules.rst?plain=1#L6-L8
- https://fmf.readthedocs.io/en/latest/modules.html#fmf.filter

I don't know why it didn't work when I've tried it :thinking: